src/app/electra/reducer.ts

```diff
@@ -27,6 +27,23 @@ export default function electraReducer(state = initialState, action: any) {
         ...state
       }
     }
+    case ActionNames.GET_GLOBAL_BALANCE_SUCCESS: {
+      return {
```

Comment:
why do we copy attributes of electraJs to reducer?. we cannot simply just access the attributes directly?